pkg/bpf/bpffs_linux.go

```diff
@@ -130,6 +131,16 @@ var (
 
 // mountFS mounts the BPFFS filesystem into the desired mapRoot directory.
 func mountFS() error {
+	if k8s.IsEnabled() {
+		log.Warning("================================= WARNING ==========================================")
```

Comment:
Nice, this will hopefully reduce confusion for people who aren't mounting it correctly.